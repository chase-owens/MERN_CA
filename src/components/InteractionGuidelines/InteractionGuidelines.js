import React from 'react';
import { theme } from '../../styles/theme';

import { isMobile } from 'react-device-detect';
import withWidth from '@material-ui/core/withWidth';

import Grid from '@material-ui/core/Grid';
import Typography from '@material-ui/core/Typography';

import { withStyles } from '@material-ui/core/styles';
const styles = theme => ({
  text: {
    fontSize: '1em'
  }
});

const InteractionGuidelines = ({ classes }) => {
  return (
    <div
      style={{
        padding: 30,
        background: theme.palette.ternary.main
      }}
    >
      <Typography
        className={classes.text}
        color='textSecondary'
        paragraph
        variant='headline'
        align='center'
      >
        Interaction Guidelines
      </Typography>
      <br />
      {/* <div
        style={{
          width: 40,
          height: 40,
          borderRadius: '50%',
          background: theme.palette.primary.main,
          margin: 'auto',
          marginTop: 5,
          marginBottom: 5
        }}
      >
        <p
          style={{
            lineHeight: 1,
            width: '100%',
            textAlign: 'center',
            lineHeight: 2.5
          }}
        >
          1
        </p>
      </div> */}
      <Typography
        className={classes.text}
        color='textSecondary'
        paragraph
        align='center'
      >
        When the dog approachees, pet the dog with one hand as long as all four
        feet are on the ground
      </Typography>
      <br />
      {/* <div
        style={{
          width: 40,
          height: 40,
          borderRadius: '50%',
          background: theme.palette.primary.main,
          margin: 'auto',
          marginTop: 5,
          marginBottom: 5
        }}
      >
        <p
          style={{
            lineHeight: 1,
            width: '100%',
            textAlign: 'center',
            lineHeight: 2.5
          }}
        >
          2
        </p>
      </div> */}
      <Typography
        className={classes.text}
        color='textSecondary'
        paragraph
        align='center'
      >
        If any excited behaviors occur or the dog leaves, discontinue petting
      </Typography>
      {/* <div
        style={{
          width: 40,
          height: 40,
          borderRadius: '50%',
          background: theme.palette.primary.main,
          margin: 'auto',
          marginTop: 5,
          marginBottom: 5
        }}
      >
        <p
          style={{
            lineHeight: 1,
            width: '100%',
            textAlign: 'center',
            lineHeight: 2.5
          }}
        >
          3
        </p>
      </div> */}
      <br />
      <Typography
        className={classes.text}
        color='textSecondary'
        paragraph
        align='center'
      >
        When the dog returns or the excited behaviors are no longer occurring,
        resume petting
      </Typography>
      {/* <div
        style={{
          width: 40,
          height: 40,
          borderRadius: '50%',
          background: theme.palette.primary.main,
          margin: 'auto',
          marginTop: 5,
          marginBottom: 5
        }}
      >
        <p
          style={{
            lineHeight: 1,
            width: '100%',
            textAlign: 'center',
            lineHeight: 2.5
          }}
        >
          4
        </p>
        
      </div> */}
      <br />
      <Typography className={classes.text} color='textSecondary' align='center'>
        If sitting or lying down occurs, pet with two hands
      </Typography>
    </div>
  );
};

export default withStyles(styles)(
  withWidth({ withTheme: true })(InteractionGuidelines)
);
