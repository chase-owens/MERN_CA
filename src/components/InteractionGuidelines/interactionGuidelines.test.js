import React from 'react';
import InteractionGuidelines from './InteractionGuidelines';
import { shallow } from 'enzyme';

describe('InteractionGuidelines', () => {
  it('renders without crashing', () => {
    const interactionGuidelines = shallow(<InteractionGuidelines />);
    expect(interactionGuidelines).toMatchSnapshot();
  });
});
