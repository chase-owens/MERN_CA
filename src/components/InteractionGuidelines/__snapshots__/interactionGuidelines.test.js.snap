// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InteractionGuidelines renders without crashing 1`] = `
<WithTheme(WithWidth(InteractionGuidelines))
  classes={
    Object {
      "text": "WithTheme-WithWidth-InteractionGuidelines---text-1",
    }
  }
/>
`;
